<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Blogs</title>
    <link href="styles.css" rel="stylesheet" />
</head>

<body>
    <nav class="Natvigation Container">
        <div class="Nav-Brand">Dhananjay Sharma
            <!--(This is Navigation Tag & is Called Nav Tag ) -->
        </div>
        <ul class="list nonbullit Nav-Pills">
            <li class="list-item-inline">
                <a class="link " href="/">Home</a>
            </li>
            <li class="list-item-inline">
                <a class="link " href="/Projects.html">Projects</a>
            </li>
            <li class="list-item-inline">
                <a class="link Link-Active" co href="/Blogs.html">Blogs</a>
            </li>
            <li class="list-item-inline">
                <a class="link" co href="/Dj_Dhanoo.html">Dj.Dhanoo</a>
            </li>
        </ul>
    </nav>
    <section class="Section of">
        <div class="container container-center">
            <h1>Hot spots localization in proteins by optimized short time Ramanujan Fourier transform</h1>
            <p>PMID: 33819134</p>
            <p>Specific functions in biological processes are dependent on protein-protein interactions. Hot spot residues
                play a key role in the determination of these interactions and have wide applications in engineering
                proteins and drug discovery. Experimental techniques to identify hotspots are often labor intensive and
                expensive. Also, most of the computational methods which have been developed are structure based and need
                some training. In this work, hotspots have been identified by sequence information alone using the Resonant
                Recognition Model (RRM). The proposed method uses characteristic period in place of traditionally used
                characteristic frequency by RRM-based methods. The characteristic period has been extracted from the
                consensus spectrum of protein families using the Ramanujan Fourier Transform (RFT). Position-period plots
                for proteins have been generated using Short Time RFT (ST-RFT) with a Gaussian window. Hot spots have been
                identified by thresholding of the signal corresponding to the protein's characteristic period in the ST-RFT.
                To enhance the performance of the ST-RFT, Gaussian window shape parameter has been optimized using
                concentration measure as a metric. Better sensitivity of this method has been observed compared to other
                reported RRM-based methods. Since the method is model independent it does not requires any training and can
                be readily used for any protein sequence provided its interface residues and protein family are known.</p>
        </div>
    </section>
    <section class="Section">
        <div class="container container-center">
            <h1>Identification of Short Exons Disunited by a Short Intron in Eukaryotic DNA Regions</h1>
            <p>PMID: 30794188</p>
            <P>Weak codon bias in short exons and separation by a short intron induces difficulty in extracting period-3
                component that marks the presence of exonic regions. The annotation task of such short exons has been
                addressed in the proposed model independent signal processing based method with following features: (a) DNA
                sequences have been mapped using multiple mapping schemes, (b) period-3 spectrums corresponding to multiple
                mappings have been optimized to enhance short exon-short intron discrimination, and (c) spectrums
                corresponding to multiple mapping schemes have been subjected to Principal Component Analysis (PCA) for
                identifying greater number of such short exons. A comparative study with other methods indicates improved
                detection of contiguous short exons disunited by a short intron. Apart from the annotation of exonic and
                intronic regions, the proposed algorithm can also complement the methods for the detection of alternative
                splicing by intron retention, as one of the characteristic feature for intron retention is the presence of
                two short exons flanking a short intron.</P>
        </div>
    </section>
    <footer class="footer">
        <div class="footer-header">Social Media Presence</div>
        <ul class="nonbullit">
            <li class="list-item-inline">
                <a class="link" href="https://github.com/xenophiliamee">
                    GitHub
                </a>
            </li>
            <li class="list-item-inline">
                <a class="link" href="https://twitter.com/Dhananj68148058?t=cBPR39GAg_yi6SI326bIAw&s=09">
                    Twitter
                </a>
            </li>
            <li class="list-item-inline">
                <a class="link" href="https://www.linkedin.com/in/dhananjay-sharma-3b209a203">
                    Linkedln
                </a>
            </li>
        </ul>
    </footer>
</body>


</html>